node/pkg/vaa/structs.go

```diff
@@ -415,6 +426,65 @@ func (v *VAA) AddSignature(key *ecdsa.PrivateKey, index uint8) {
 	})
 }
 
+func (v *VAA) IsTransfer() bool {
+	return len(v.Payload) > 0 && (v.Payload[0] == 1) || (v.Payload[0] == 3)
```

Comment:
Not sure of the context but you probably need to check that the emitter chain + contract is a token bridge. 🤔 not sure if we _explicitly_ have that available, but we could make it so.

Reply:
In my case, I will not get this far if it's not from the token bridge. I don't want to add the complexity of making that check here (since it would mean listing all of the token bridge addresses just for that). Instead, I just added a comment saying that the caller should ensure this is only called for a VAA from the token bridge.